lib/services/pdf_renderer/renderer.js

```diff
@@ -13,7 +13,7 @@ const RENDERERS = { shape, text, image }
 export function renderItem(doc, game, component, item, parentDimensions, index, total) {
   // get layers for this component
   const template = store.getters.findTemplate(component.templateId),
-    layers = store.getters.findAllTemplateLayers(template),
+    layers = store.getters.findAllTemplateLayers(template).filter(l => l.hide == undefined || !l.hide),
```

Comment:
Here I want to push this filtering logic down into the store getters, and have this refer to a nice method name, like `findAllVisibleTemplateLayers`.

Reply:
You're completely right, that was a bit lazy from my side, sorry! Moved it there.